lib/perl/Genome/Model/SomaticValidation/Command/ValidateSvs/CreateAssembledContigReference.t

```diff
@@ -21,7 +22,9 @@ use_ok($pkg) or die();
 
 my $test_dir = Genome::Utility::Test->data_dir_ok($pkg);
 
+my $anp = Genome::Test::Factory::AnalysisProject->setup_object();
```

Comment:
Nope; this test doesn't depend on the system AnP.

The `SomaticValidation` build passes along its model's AnP when creating the reference for realignment, so it will belong to the specific project for which it was created instead of the generic system.